### PR TITLE
update demo to 3scale 2.6

### PR DESCRIPTION
**Pull request by hguerrero**
##### SUMMARY

Updated 3scale version to 2.6.0 GA

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

3scale security demo workload

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: ansible/roles/ocp-workload-3scale-demo/tasks/workload.yml [hunk 1]
```diff
@@ -88,22 +88,15 @@
   shell: "oc project default"
 
 - name: Create the 3scale app
-  shell: "oc new-app -f https://raw.githubusercontent.com/3scale/3scale-amp-openshift-templates/{{threescale_version}}/amp/amp-eval-tech-preview.yml \
+  shell: "oc new-app -f https://raw.githubusercontent.com/jbossdemocentral/3scale-security-oidc-demo/master/support/templates/3scale-amp-template.yml \
```
Comment (fridim) [inline ansible/roles/ocp-workload-3scale-demo/tasks/workload.yml]: Can you please pin the versions ?
Here you are using the master branch. 

Either use a tag, or a commit ID

File: ansible/roles/ocp-workload-3scale-demo/tasks/workload.yml [hunk 2]
```diff
@@ -88,22 +88,15 @@
...
 
-- template:
-    src: wildcard-route.yml.j2
-    dest: /tmp/wildcard-route.yml
-
-- name: Create wildcard route
-  shell: oc create -f /tmp/wildcard-route.yml -n {{ threescale_project }}
-  register: create_wildcard_route
+- name: Create new imagestreams in openshift namespace
+  shell: "oc apply -n openshift -f https://raw.githubusercontent.com/jbossdemocentral/3scale-security-oidc-demo/master/support/templates/3scale-image-streams.yml"
```
Comment (fridim) [inline ansible/roles/ocp-workload-3scale-demo/tasks/workload.yml]: Same here, please pin the version

File: ansible/roles/ocp-workload-3scale-demo/tasks/workload.yml (hunk 1 above)
Comment (hguerrero) [reply ansible/roles/ocp-workload-3scale-demo/tasks/workload.yml]: tagged repo and use the same var as before

File: ansible/roles/ocp-workload-3scale-demo/tasks/workload.yml (hunk 2 above)
Comment (hguerrero) [reply ansible/roles/ocp-workload-3scale-demo/tasks/workload.yml]: same as above

**Pull request merged**